README.md

```diff
@@ -19,3 +19,18 @@ You can then install the development requirements using
 and run the tests with
 
     pipenv run pytest
+
+How to reprovision certificates
+-------------------------------
+
+If a certificate wasn't automatically renewed for some reason, you can trigger a renewal in this way:
+
+1. Log in to the `cert-manager` server as root and run `cerbot delete`, then select the number of the certificate you want to delete.
+
+2. From the consul web interface, navigate to `ocim/certs` and delete the entry corresponding to the old certificate.
```

Comment:
@clemente Thanks a lot for adding this info! 💯 

A quick drive-by comment: Could you add a link to the "consul web interface" (even if it's just an example)?

Reply:
@itsjeyd Done in the new PR.